easybuild/easyconfigs/g/GenomeWorks/GenomeWorks-2021-02-2_dep_versions_n_file_extension.patch

```diff
@@ -0,0 +1,46 @@
+# author: Denis Kristak (INUITS)
+# set correct dependecy versions && expect *.a files to copy, rather than *.so
+diff -ruN GenomeWorks-dev_orig/pygenomeworks/requirements.txt GenomeWorks-dev/pygenomeworks/requirements.txt
+--- GenomeWorks-dev_orig/pygenomeworks/requirements.txt	2021-06-08 19:42:06.199086912 +0200
++++ GenomeWorks-dev/pygenomeworks/requirements.txt	2021-06-08 19:41:42.299168164 +0200
+@@ -16,12 +16,12 @@
+ 
+ 
+ 
+-Cython~=0.29.14
++Cython==0.29.21
+ intervaltree==3.1.0
+-matplotlib==3.3.2
+-networkx==2.4
+-numpy~=1.17.5
+-pytest==4.4.1
++matplotlib==3.3.3
++networkx==2.5.1
++numpy==1.19.4
++pytest==6.2.4
```

Comment:
This is going to be annoying to maintain, why not use a `sed` command in the easyconfig to transform the `==` to `>=` instead (less maintenance going forward)

Reply:
@boegel whether i put the sed in preconfigopts of the main file or in the first extension, it does not work, maybe pip is doing some magic idk about (copying files before anything else happens) but i've tried what you're proposing multiple times and it did now work

Reply:
You need to take into account that when the `genomeworks` extensions (Python package) is installed, that's done in a fresh unpacked source directory, so you need to make sure the `sed` commands are used there.

I've fixed that in 142f9f7 by adding the necessary `sed` command to `local_genomeworks_preinstallopts`